modules/gapi/test/streaming/gapi_streaming_vpl_core_test.cpp

```diff
@@ -194,11 +195,11 @@ TEST(OneVPL_Source_Surface, InitSurface)
     // check self consistency
     EXPECT_EQ(reinterpret_cast<void*>(surf->get_handle()),
               reinterpret_cast<void*>(mfx_core_handle));
-    EXPECT_EQ(0, surf->get_locks_count());
-    EXPECT_EQ(0, surf->obtain_lock());
-    EXPECT_EQ(1, surf->get_locks_count());
-    EXPECT_EQ(1, surf->release_lock());
-    EXPECT_EQ(0, surf->get_locks_count());
+    EXPECT_TRUE(0 == surf->get_locks_count());
```

Comment:
Why is not EXPECT_EQ?

Reply:
because o is int_t by consideration but get_lock_count returns size_t and gcc finds it inappropriate 
P.S. 0UL and 0ULL cannot be aligned to size_t on different platforms together